libbeat/processors/add_cloud_metadata/provider_google_gce.go

```diff
@@ -20,11 +20,25 @@ package add_cloud_metadata
 import (
 	"path"
 
+	"gopkg.in/yaml.v2"
+
 	"github.com/elastic/beats/v7/libbeat/common"
 	s "github.com/elastic/beats/v7/libbeat/common/schema"
 	c "github.com/elastic/beats/v7/libbeat/common/schema/mapstriface"
 )
 
+type KubeConfig struct {
+	Clusters []Clusters `yaml:"clusters"`
+}
+
+type Clusters struct {
+	Cluster Cluster `yaml:"cluster"`
+}
+
+type Cluster struct {
```

Comment:
Should this be server instead? and then we can change type `Clusters` to Cluster?